src/test/java/apoc/schema/SchemasTest.java

```diff
@@ -270,7 +270,46 @@ public void testIndexes() {
             assertEquals(new Double(1), r.get("valuesSelectivity"));
             assertEquals("Index( GENERAL, :Foo(bar) )", r.get("userDescription"));
 
-            assertTrue(!result.hasNext());
+            assertFalse("Should not have another row",result.hasNext());
+        });
+    }
+
+    @Test
+    public void testIndexesByLabel() {
+        db.execute("CREATE INDEX ON :Foo(bar)").close();
+        db.execute("CREATE INDEX ON :Bar(foo)").close();
+        try (Transaction tx = db.beginTx()) {
+            db.schema().awaitIndexesOnline(5, TimeUnit.SECONDS);
+            tx.success();
+        }
+        testResult(db, "CALL apoc.schema.nodes('Foo')", (result) -> {
```

Comment:
usare `testCall` e togliere questa riga `assertFalse("Should not have another row",result.hasNext());`
```suggestion
        testCall(db, "CALL apoc.schema.nodes('Foo')", (r) -> {
            assertEquals(":Foo(bar)", r.get("name"));
```

Reply:
Fare così anche negli altri test